src/library_signals.js

```diff
@@ -138,20 +138,16 @@ var funs = {
     return -1;
   },
 #if SUPPORT_LONGJMP
-#if ASSERTIONS
-  siglongjmp__deps: ['longjmp'],
```

Comment:
What is this `__deps` and how is this different from entries in deps_info.py? I deleted this because tests run correctly without this, but not sure what this is for.